charts/prometheus-snmp-exporter/templates/servicemonitor.yaml

```diff
@@ -22,6 +22,8 @@ spec:
     scrapeTimeout: {{ .scrapeTimeout | default $.Values.serviceMonitor.scrapeTimeout }}
     params:
       module: {{ .module | default $.Values.serviceMonitor.module }}
+      auth:
+      - {{ .auth | default $.Values.serviceMonitor.auth }}
```

Comment:
Sorry only just noticed, there is already an `auth` field referenced on lines 29-31, could you remove those in favour of this please.